packages/developer-portal/src/core/index.tsx

```diff
@@ -5,10 +5,6 @@ import { render } from 'react-dom'
 import ReactGA from 'react-ga'
```

Comment:
The run function needs to be refactored to lazy load the App component. See: https://github.com/reapit/foundations/blob/master/packages/aml-checklist/src/core/index.tsx#L61